### PR TITLE
Temp dev

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
I have worked on fixes and readability of video Library
- Fixed video Card sizes
- Home page cards and their accessibility
- added clear watch later videos
- added clear liked videos
- added private and public routes
- fixed error in playlist videos
- fixed styling issues with buttons

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -14,10 +14,10 @@ export default function Navbar() {
           <h1>FullStackDevelopement</h1>
         </Link>
       </div>
-      <div className="search-bar">
+      {/* <div className="search-bar">
         <input placeholder="Search Video" />
```
Comment (ankit9015) [inline src/components/Navbar.jsx:18]: Remove commented code if not needed

**Review by neetukumari4858 — APPROVED** [review]
The folder is well Structured 
code is readable

**Review by ankit9015 — APPROVED** [review]
Overall code is well structured and good to merge

File: src/components/Navbar.jsx (hunk 1 above)
Comment (Kothalkarpriya) [reply src/components/Navbar.jsx:18]: Yes I have to work on it next

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Link, NavLink } from "react-router-dom";
 import "../assests/styles/navbar.css";
-import { FaSearch } from "react-icons/fa";
+// import { FaSearch } from "react-icons/fa";
```
Comment (t007rushi) [inline src/components/Navbar.jsx:4]: Remove if not required

File: src/components/VideoCard.jsx [hunk 3]
```diff
@@ -19,7 +19,8 @@ export default function VideoCard({ videoItem }) {
           />
         </div>
         <div className="text">
-          <h4 className="title">{videoItem.title}</h4>
+          <p className="title">{videoItem.title}</p>
+          <p className="title ">Creator: {videoItem.creator}</p>
```
Comment (t007rushi) [inline src/components/VideoCard.jsx:23]: Destructure them

File: src/route/RoutePath.jsx [hunk 4]
```diff
@@ -13,21 +13,64 @@ import {
...
+          </RouteAuth>
+        }
+      />
       <Route path="/VideoCardPage" element={<VideoCardPage />} />
-      <Route path="/Playlist/:listId" element={<PlaylistPage />} />
-      <Route path="/Modal" element={<Modal />} />
+      <Route
+        path="/Playlist/:listId"
+        element={
+          <RouteAuth>
```
Comment (t007rushi) [inline src/route/RoutePath.jsx:61]: Rename it to ```RequireAuth``` as per convention

**Review by t007rushi — APPROVED** [review]
With few suggestions to look for
Go ahead
Files structure is well maintained

File: src/route/RoutePath.jsx (hunk 4 above)
Comment (Kothalkarpriya) [reply src/route/RoutePath.jsx:61]: OK I will do it next PR

File: src/components/Navbar.jsx (hunk 2 above)
Comment (Kothalkarpriya) [reply src/components/Navbar.jsx:4]: Yes I will be working on search in next PR

**Pull request merged**